tests/configs/configs_test.go

```diff
@@ -43,6 +43,13 @@ var _ = Describe("Configs", func() {
 		command.Wait()
 		print(" [done]\n")
 
+		print("enabling metrics server...")
+		command = exec.Command("sed", "-i", "s/enable_metrics_server: false/enable_metrics_server: true/g", "/tmp/dummy-domain-values.yml")
```

Comment:
Hmm, I don't love this but it makes sense. Should we maybe do this only for the 1 relevant/required config-optional file?

Reply:
You mean just for the "check optional configs" test? Or a new section?